pom.xml

```diff
@@ -4,19 +4,18 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>4.40</version>
+        <version>4.82</version>
         <relativePath />
     </parent>
     <groupId>io.jenkins.plugins</groupId>
     <artifactId>zohosprints</artifactId>
-    <version>2.1-SNAPSHOT</version>
+    <version>${revision}${changelist}</version>
```

Comment:
Same like below and in the scm block, working with placeholders injected by the maven release plugin is always preferred, over configuring these entries manually.